packages/components/src/components/hds/form/super-select/multiple/base.js

```diff
@@ -0,0 +1,133 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import PowerSelectComponent from 'ember-power-select/components/power-select';
+import anchoredPositionModifier from '../../../../../modifiers/hds-anchored-position';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+
+const DEFAULT_HORIZONTAL_POSITION = 'bottom-start';
+const HORIZONTAL_POSITION_MAPPING = {
+  left: 'bottom-start',
+  center: 'bottom',
+  right: 'bottom-end',
+};
+
+export default class HdsSuperSelectMultipleBaseComponent extends PowerSelectComponent {
+  @tracked powerSelectAPI;
+  @tracked showOnlySelected = false;
+  @tracked showNoSelectedMessage = false;
+
+  get selectedCount() {
+    return this.selected?.length || '0';
+  }
+
+  get optionsCount() {
+    return this.options?.length || '0';
+  }
+
+  get resultCountMessage() {
+    return `${this.selectedCount} selected of ${this.optionsCount} total`;
+  }
+
+  @action calculatePosition(trigger, content) {
+    // use `hds-anchored-position` to calculate and set position
+    anchoredPositionModifier(content, [trigger], {
+      placement: this.args.horizontalPosition
+        ? HORIZONTAL_POSITION_MAPPING[this.args.horizontalPosition]
+        : DEFAULT_HORIZONTAL_POSITION,
+      offsetOptions: 4,
+      enableCollisionDetection: true,
+    });
+    // prevent PowerSelect from setting position
+    return {};
+  }
+
+  @action
+  setPowerSelectAPI(powerSelectAPI) {
```

Comment:
should we add a comment here and explain why we need this (and why it's different from the "single" variant, where the `registerAPI` is simply forwarded to the PowerSelect)?

Reply:
@alex-ju I believe you worked on this part so would you mind adding an explanatory comment?

Reply:
Added in a58dcd8